halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/problem/v1/ConfigProblemSetBuilder.java

```diff
@@ -32,9 +34,16 @@
 public class ConfigProblemSetBuilder {
   private List<ConfigProblemBuilder> builders = new ArrayList<>();
 
+  @Getter
```

Comment:
It defaults to public. Everything in the core/config code has public getters & setters, it's only the CLI package where I need to be careful to delineate what to generate.